src/libxdata/XData.cpp

```diff
@@ -69,20 +68,27 @@ void XData::cancelLoading() {
 void XData::loadAirports() {
     const AirportLoader loader(world);
 
-    //Custom scenery
+    try {
+        loadCustomScenery(loader);
+    } catch (const std::exception &e) {
+        logger::warn("Unable to parse custom scenery. Check your scenery_packs.ini file.", e.what());

```

Comment:
Stack corruption due to missing %s placeholder

Reply:
Ah damn sorry man. Ok I fixed that. Where did you see that warning by the way? My compiler didnt pick that up.

Reply:
I only saw it in the manual review.